client/package.json

```diff
@@ -20,14 +20,16 @@
     "react-dom": "18.0.0",
     "react-scroll": "^1.8.7",
     "react-slick": "^0.29.0",
-    "slick-carousel": "^1.8.1"
+    "slick-carousel": "^1.8.1",
+    "styled-components": "^6.1.11"
   },
   "devDependencies": {
-    "@types/node": "17.0.25",
-    "@types/react": "18.0.6",
-    "@types/react-dom": "18.0.2",
+    "@types/node": "^17.0.25",
+    "@types/react": "^18.3.2",
+    "@types/react-dom": "^18.3.0",
     "@types/react-scroll": "^1.8.3",
     "@types/react-slick": "^0.23.8",
+    "@types/styled-components": "^5.1.34",
```

Comment:
Can't we add style with out styled components and without adding new dependency.